### PR TITLE
Jdhzzz ct usa

### PR DESCRIPTION
**Pull request by jdhzzz**
Added a file with central CT restaurants.  Yum.  :taco: :beers:

### CODE REVIEW

File: jdhzzz-CT-USA.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Central CT
```
Comment (paluchas) [inline jdhzzz-CT-USA.md:1]: Can you add example hometown name, please? I'm overseas in the UK and have no idea what it means Central CT

File: jdhzzz-CT-USA.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+## Places to Eat
+- El Sombrero
+- Somewhere in Bangkok
+- Sax Pizza
+- Saints
+- Anthony Jacks Wood Fired Grill
+
+## Places to Visit
+
```
Comment (paluchas) [inline jdhzzz-CT-USA.md]: Can you suggest some attractions I could see near your hometown?

File: jdhzzz-CT-USA.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Central CT
+
+## Places to Eat
```
Comment (paluchas) [inline jdhzzz-CT-USA.md]: Which place is the best for a 🍺

File: jdhzzz-CT-USA.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+- El Sombrero
+- Somewhere in Bangkok
+- Sax Pizza
+- Saints
+- Anthony Jacks Wood Fired Grill
```
Comment (githubteacher) [inline jdhzzz-CT-USA.md:8]: @jdhzzz any specific recommendations on what kind of 🍕 to get from Anthony Jacks?

**Review by paluchas — COMMENTED** [review]
Please, add some more details about your place. Thx

File: jdhzzz-CT-USA.md (hunk 2 above)
Comment (githubteacher) [inline jdhzzz-CT-USA.md]: I enjoy craft beer, any breweries I should visit? 🍺

**Review by githubteacher — CHANGES_REQUESTED** [review]

File: jdhzzz-CT-USA.md (hunk 4 above)
Comment (jdhzzz) [reply jdhzzz-CT-USA.md:8]: Margherita

**Review by paluchas — APPROVED** [review]
I might try Anthony Jack where there

**Pull request merged**